automata/symbol/graph/data_root_settings.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+fallback_data_root_path = "automata-embedding-data"
+data_root_path = os.environ.get("DATA_ROOT_PATH", fallback_data_root_path)
```

Comment:
again, we shouldn't need this - relegate to the config

Reply:
Misunderstood before—fixed now!